src/state/store.ts

```diff
@@ -12,16 +12,28 @@ import type { ReduxState } from './reducer'
 import { apiSlices, reducer, slices } from './reducer'
 import { assetApi } from './slices/assetsSlice/assetsSlice'
 import { marketApi, marketData } from './slices/marketDataSlice/marketDataSlice'
-import { opportunitiesApi } from './slices/opportunitiesSlice/opportunitiesSlice'
+import { initialState, opportunitiesApi } from './slices/opportunitiesSlice/opportunitiesSlice'
 import { portfolioApi } from './slices/portfolioSlice/portfolioSlice'
 import * as selectors from './slices/selectors'
 import { txHistoryApi } from './slices/txHistorySlice/txHistorySlice'
 import { validatorDataApi } from './slices/validatorDataSlice/validatorDataSlice'
 
+const migrations = {
```

Comment:
let's pull this out to a migration directory

Reply:
then name them like `migration-000.ts`

Reply:
we could do like `migration-000-clear-opportunities.ts` then dynamically apply them in order